### DISCUSSION TITLE
GraphQL API doesn't indicate which files in a PR are new

### DISCUSSION THREAD

**Discussion by RyanCavanaugh** [community/community#24587] [Apps, API and Webhooks]
<p>The REST API would tell you if a file was added, modified or deleted: <a href="https://developer.github.com/v3/pulls/#list-pull-requests-files" rel="nofollow noopener">https://developer.github.com/v3/pulls/#list-pull-requests-files</a></p>
<p>The GraphQL API doesn’t seem to have any way to do this - you can see how many additions/deletions were in a commit/PR overall, but there isn’t a way (without separately querying the repo) to see which files are new vs simply had lines added. This is functionality we used in the REST API for some of our automation.</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/ryancavanaugh">@ryancavanaugh</a>,</p>
<p>Thanks for the feedback! I’ve taken your suggestion and passed it along to the appropriate teams.<br>
I don’t have an ETA on when or if this request will be implemented but this is in good hands.</p>

**Comment by fusion2004** [comment]
<p>Similarly, I’m running into wanting to know which files were deleted!</p>

**Comment by kevgo** [comment]
<p>The proposed solution isn’t really one. It merely admits that the requested feature is missing. It would be nice to leave this as “unresolved” and notify subscribers to when this is is actually fixed. Thanks!</p>
